modules/ballerina-core/src/main/java/org/ballerinalang/runtime/ServerConnectorMessageHandler.java

```diff
@@ -141,9 +142,11 @@ public static void handleErrorFromOutbound(Context balContext, Throwable throwab
         String errorMsg = ErrorHandlerUtils.getErrorMessage(throwable);
         String stacktrace = ErrorHandlerUtils.getServiceStackTrace(balContext, throwable);
         String errorWithTrace = errorMsg + "\n" + stacktrace;
-        log.error(errorWithTrace);
         outStream.println(errorWithTrace);
 
+        // bre log should contain bre stack trace, not the ballerina stack trace
+        breLog.error("error : " + errorMsg + ", ballerina service stack trace : " + stacktrace, throwable);
```

Comment:
Same as above, might not need to keep a space before the colon:
https://www.grammarly.com/handbook/punctuation/colon/